mainnet/run-proxy.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-docker run --network=elrond-squad --ip=${IP} --name proxy elrondnetwork/elrond-proxy:v1.1.3
+docker run --network=elrond-squad --ip=${IP} --name proxy elrondnetwork/${PROXY_TAG}
```

Comment:
From the README, the user doesn't know that it should also set `PROXY_TAG`:

```
export IP=10.0.0.2
./mainnet/run-proxy.sh
```

Perhaps add it there as well?

Of course, the user can `source .env` instead (but then we have to instruct the user to read the `.env` file carefully).

Reply:
reverted